helm/g2p-sandbox/values.yaml

```diff
@@ -520,6 +520,54 @@ ph-ee-engine:
     gradle:
       command: ./gradlew test -Dcucumber.filter.tags="@gov"
 
+  billPay:
+    enabled: true
+    image: docker.io/openmf/ph-ee-bill-pay:latest
```

Comment:
these are duplicate fields, remove the fields that are already set in ph-ee-engine values

Reply:
the image tag is different, one is tagged version and one is latest, should i still replace?